frontend/views/containers/chatroom/ChatMain.vue

```diff
@@ -754,11 +757,6 @@ export default ({
       return messageBCR.top < (conversationBCR.bottom - 10) && messageBCR.bottom > (conversationBCR.top + 10)
     },
     proximityDate,
-    chatroomHasSwitchedFrom (chatroomID) {
-      // Some async operations (eg. this.rerenderEvents) are time-consuming, and user could switch away from the chatroom until the operation is not completed.
-      // This method checks if that has happened.
```

Comment:
I wish this comment was still kept somewhere (after rephrased). Checking whether the chatroom has been switched away and stopping operations could sound obvious to us, but might not to people with fresh eyes at first. Having this will help readability.

Reply:
Noted